R/comparePlot.R

```diff
@@ -78,13 +78,13 @@
 #' @examples
 #' \dontrun{
 #'   # Retrieve the first two contrasts from a DGEobj as a list of dataframes (length = 2; named items)
-#'   ttList <- getType(DGEobj, "topTable")[1:2]
+#'   contrastList <- getType(DGEobj, "topTable")[1:2]
```

Comment:
@kar-agg-gen I think we just need to be sure that when we use `contrastList` throughout as an input, it's always the same type of object being used as an input. That seems to be what you've done here, to me.

Reply:
Yes